src/libraries/System.Text.Json/tests/Serialization/Array.ReadTests.cs

```diff
@@ -96,7 +96,7 @@ public static void ReadByteListAsJsonArray()
         [Fact]
         public static void DeserializeObjectArray_36167()
         {
-            // https://github.com/dotnet/corefx/issues/36167
+            // https://github.com/dotnet/runtime/issues/29019
```

Comment:
With issue number changed, the test names should be updated (it stills says `_36167`). We should just remove the issue number from it to avoid having to keep updating it. Same elsewhere (like `DictionaryOfObject_37569`).

Reply:
Thanks @ahsonkhan... I will follow up on a PR.